static/js/util/util.js

```diff
@@ -85,7 +110,7 @@ export function makeCourseStatusDisplay(course, now = moment()) {
 
     let verificationDate = moment(courseOrRun.verification_date);
     if (verificationDate.isAfter(now, 'day')) {
-      return <Button bsStyle="success">UPGRADE TO VERIFIED</Button>;
+      return <Button bsStyle="success" onClick={onUpgradeBtnClick}>UPGRADE TO VERIFIED</Button>;
```

Comment:
I don't think you need `onClick` here at all, you should be able to just use `href` and `target` here